content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -170,3 +170,29 @@ perform security-sensitive operations such as application creation or modificati
 As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), the German version of the Analytics Builder documentation,
 which is available as a separate webhelp until release 10.17, is no longer provided.
 See also the information on the restructured *Streaming Analytics guide* in the [Streaming Analytics](/release-10-18-0/streaming-analytics-10-18-0/) release notes.
+
+##### Cumulocity IoT transport in Apama
+
+As of Apama 10.15.3, the Cumulocity IoT transport supports handling of standard Cumulocity IoT microservice proxy environment variables such as
```

Comment:
I don't think this is worth calling out and also there were some changes that are required on the PAQ side which didn't go in to 10.18